department-of-veterans-affairs/caseflow
# Connect frontend to eFolder

Connects: #2340

To integrate with eFolder Express the Caseflow client needs to request documents from the eFolder Express backend. This PR introduces a feature hidden behind the `efolder_docs_api` feature flag that pulls document content from eFolder Express.

**Test Plan**
- Run Caseflow on port 3000, and eFolder on port 4000. Don't enable the feature flag and make sure we get the documents from Caseflow.
- Run Caseflow on port 3000, and eFolder on port 4000. Enable the feature flag and make sure we get the documents from eFolder.
    - Enable the feature flag by executing `FeatureToggle.enable!(:efolder_docs_api)` in the rails console for Caseflow
